test/e2e/data/infrastructure-provider-byoh/v1beta1/templates/vm/cluster-template-topology.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       - "192.168.0.0/16"
     serviceDomain: "cluster.local"
   topology:
-    class: ${CLUSTER_CLASS_NAME}
+    class: quickstart
```

Comment:
Suggestion: if these changes are not related, it could have been a separate PR.

Reply:
Sure 👍